package.json

```diff
@@ -76,7 +76,7 @@
     "ioredis": "^4.27.6",
     "js-yaml": "^4.1.0",
     "jsonwebtoken": "^8.5.1",
-    "lightning": "^5.4.1",
+    "lightning": "^5.8.3",
```

Comment:
to avoid future conflicts with yarn.lock file please remove `yarn.lock` and re-create it again with `yarn install`